target_chains/ethereum/contracts/contracts/pyth/PythAccumulator.sol

```diff
@@ -134,35 +107,24 @@ abstract contract PythAccumulator is PythGetters, PythSetters, AbstractPyth {
         }
     }
 
-    function extractPriceInfosFromWormholeMerkle(
-        bytes memory encoded
+    function extractPriceInfoFromWormholeMerkle(
```

Comment:
this function seems redundant -- it's identical to the one it calls in the body ?

Reply:
fixed ty!